docs/core-protocol/public-lock/transfers.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Transfers
+description: The Public Lock contract implements the ERC721 specification, including transfer functions, but their behavior can be customized!
+sidebar_position: 4
+---
+
+The Public Lock contract implements the [ERC721 specification](https://docs.openzeppelin.com/contracts/2.x/api/token/erc721), including transfer functions (`transferFrom`, `safeTransferFrom`, and the related `approve` and `setApprovalForAll`), but their behavior can be customized.
+
+:::info
+Expired keys **can not** be transferred.
+:::
+
+## Key Managers
```

Comment:
@julien51 perhaps before this `key managers` section, also have a `lock managers` section? unless someone is deep in the lingo, they won't know what a lock manager is